### PR TITLE
Add starsessions library to the 3rd-party list.

### PR DESCRIPTION
**Pull request by alex-oleshkevich**
This PR adds [starsessions](https://github.com/alex-oleshkevich/starsessions) library to the list of Starlette plugins.

Relates to https://github.com/encode/starlette/pull/499

### CODE REVIEW

File: docs/third-party-packages.md [hunk 1]
```diff
@@ -90,6 +90,14 @@ It relies solely on an auth provider to issue access and/or id tokens to clients
 Middleware for Starlette that allows you to store and access the context data of a request.
 Can be used with logging so logs automatically use request headers such as x-request-id or x-correlation-id.
 
+
+### Starsessions
+
+<a href="https://github.com/alex-oleshkevich/starsessions" target="_blank">GitHub</a>
+
+An alternate session support implementation. With customizable storage backends.
```
Comment (Kludex) [inline docs/third-party-packages.md]:
```suggestion
An alternate session support implementation with customizable storage backends.
```

**Review by Kludex — CHANGES_REQUESTED** [review]
Thanks for the PR @alex-oleshkevich ! :tada: 

Would you mind rebasing it?

**Comment by alex-oleshkevich** [comment]
Updated.

**Review by Kludex — APPROVED** [review]
Thanks @alex-oleshkevich ! :tada:

**Pull request merged**